internal/apps/user/restrict.go

```diff
@@ -50,6 +49,20 @@ func Restrict(r restricter.HasPermer) restricter.ElementFunc {
 			return filter(element, littleDataFields)
 		}
 
+		// get the users `vote_delegated_from_users_id`
+		var requestUser struct {
+			VoteDelegationIds []int `json:"vote_delegated_from_users_id"`
+		}
+		if err := r.Get("users/user", uid, &requestUser); err != nil {
```

Comment:
This is the only place where the resticter uses the `Get()`-Method. I try to have the interfaces as small as possible.

I don't understand the `vote_delegated_from_users_id` field. Can you explain to me what it does and also, why it is impossible to find a solution without the `Get()` method?

If it is really necessary to get the data of the request_user, then maybe it would be better to change the signature of the function. The first argument could be an user-object instead of an user-id. If more restricts need the user-data, then it has only to be decoded once.

Reply:
The get method is also used in polls. The `vote_delegated_from_users_id` is relation to other users, which are "Vollmachtgeber". There might be a solution without `Get`, but this was the solution not to change every restricter signature and leave the possibility open to access more cache data if needed.

Reply:
Ok.

For this repo it is ok for me, since I see it only as a temporary solution until we have OpenSlides4. If we get similar problems in OS4, then I would like to find a better solution.